wolfssl/wolfcrypt/aes.h

```diff
@@ -161,6 +165,10 @@ typedef struct Aes {
     int rdFd; /* socket to read from */
     struct msghdr msg;
     int dir;  /* flag for encrpyt or decrypt */
+#endif
+#if defined(WOLFSSL_DEVCRYPTO_AES) || defined(WOLFSSL_DEVCRYPTO_CBC)
```

Comment:
For cases where not building with ./configure can you also check that `WOLFSSL_DEVCRYPTO` is defined?